<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadH5EbsdWidget</class>
 <widget class="QWidget" name="ReadH5EbsdWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>335</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>EBSD HDF5 File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFSDropLineEdit" name="m_InputFile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_InputFileBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>4</number>
     </property>
     <item row="1" column="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <layout class="QFormLayout" name="formLayout_4">
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
         </property>
         <property name="text">
          <string>X Res:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="m_XRes">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
         </property>
         <property name="text">
          <string>--</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_17">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
         </property>
         <property name="text">
          <string>Y Res:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="m_YRes">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
         </property>
         <property name="text">
          <string>--</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_200">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
         </property>
         <property name="text">
          <string>Z Res:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="m_ZRes">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
         </property>
         <property name="text">
          <string>--</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
         </property>
         <property name="text">
          <string>X Dim:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="m_XDim">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
         </property>
         <property name="text">
          <string>--</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
         </property>
         <property name="text">
          <string>Y Dim:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="m_YDim">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
         </property>
         <property name="text">
          <string>--</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
         </property>
         <property name="text">
          <string>Z Dim:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="m_ZDim">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
         </property>
         <property name="text">
          <string>--</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="4">
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_23">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
         </property>
         <property name="text">
          <string>EBSD Manufacturer:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="m_EbsdManufacturer">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
         </property>
         <property name="text">
          <string>XXX</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
         </property>
         <property name="text">
          <string>Sample Transformation (Recommended):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="m_SampleTransformationLabel">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
         </property>
         <property name="text">
          <string>false</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_20">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
         </property>
         <property name="text">
          <string>Euler Transformation (Recommended):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="m_EulerTransformationLabel">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
         </property>
         <property name="text">
          <string>false</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::FieldsStayAtSizeHint</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item row="0" column="1">
        <widget class="QLabel" name="m_ZMin">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
         </property>
         <property name="text">
          <string>--</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_54">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
         </property>
         <property name="text">
          <string>Maximum Slice:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="m_ZMax">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
         </property>
         <property name="text">
          <string>--</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
         </property>
         <property name="text">
          <string>Minimum Slice:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="m_RefFrameZDir">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
         </property>
         <property name="text">
          <string>Unknown</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_24">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
         </property>
         <property name="text">
          <string>Stacking Order:</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="widgetLayout">
     <item row="2" column="2">
      <widget class="QSpinBox" name="m_ZEndIndex">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>End Slice</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" rowspan="5">
      <widget class="QListWidget" name="m_CellList">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="m_DataArraysCheckBox">
       <property name="text">
        <string>Data Arrays to Read</string>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QCheckBox" name="m_UseTransformations">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Use Recommended Transformations</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="m_ZStartIndex">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Start Slice</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QComboBox" name="m_AngleRepresentationCB">
       <item>
        <property name="text">
         <string>Radians</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Degrees</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string extracomment="Are the Euler angles represented in Degrees or Radians"/>
       </property>
       <property name="text">
        <string>Euler Representation</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFSDropLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QFSDropLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
